front/app/modules/commercial/machine_translations/citizen/components/PostShowTranslatedCommentBody.tsx

```diff
@@ -17,23 +18,20 @@ interface Props {
 const PostShowTranslatedCommentBody = ({
   translateButtonClicked,
   commentContent,
-  locale,
   commentId,
 }: Props) => {
-  const isMachineTranslationsEnabled = useFeatureFlag({
-    name: 'machine_translations',
-  });
```

Comment:
We already check for this feature flag in the general "useTranslation" hook, so we ca remove it here.